docs/commands.rst

```diff
@@ -608,49 +613,66 @@ Here is an example command to covert a job run XML into JUnit results.
 
  xsltproc job2junit.xml simple_job.07/job.xml > simple_job.07/junit.xml
 
+.. _legacy_rhts_cmds:
+
 Legacy RHTS Commands
```

Comment:
I would move the whole RHTS section, and every reference to it, to a dedicated page. Like the page for removing RHTS, for example.

Reply:
I believe I've done this already.  This is just a brief snippet to guide people to that section with the detals.